src/Tizen.NUI.Components/Controls/Slider.cs

```diff
@@ -575,6 +580,123 @@ public uint TrackThickness
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public float WarningStartValue
+        {
+            get
+            {
+                return warningStartValue;
+            }
+            set
+            {
+                warningStartValue = value;
+                UpdateValue(); // Seoyeon : Maybe should update..
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the color of the warning track image object.
+        /// </summary>
+        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Color WarningTrackColor
+        {
+            get
+            {
+                return warningTrackImage?.BackgroundColor;
+            }
+            set
+            {
+                if (null != warningTrackImage)
+                {
+                    warningTrackImage.BackgroundColor = value;
+                    sliderStyle.WarningTrack.BackgroundColor = value;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the color of the warning slided track image object.
+        /// </summary>
+        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Color WarningSlidedTrackColor
+        {
+            get
+            {
+                return warningSlidedTrackImage?.BackgroundColor;
+            }
+            set
+            {
+                if (null != warningSlidedTrackImage)
+                {
+                    warningSlidedTrackImage.BackgroundColor = value;
+                    sliderStyle.WarningProgress.BackgroundColor = value;
+                }
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the resource url of the warning thumb image object.
+        /// </summary>
+        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string WarningThumbImageUrl
+        {
+            get
+            {
+                return warningThumbImageUrl;
+            }
+            set
+            {
+                warningThumbImageUrl = value;
+            }
+        }
+
+        /// <summary>
+        /// Gets or sets the resource url selector of the warning thumb image object.
+        /// Getter returns copied selector value if exist, null otherwise.
+        /// </summary>
+        /// <exception cref="NullReferenceException">Thrown when setting null value.</exception>
+        /// This will be public opened later after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public StringSelector WarningThumbImageUrlSelector
```

Comment:
Please use `Selector<string>` instead of `StringSelector`.
`Selector<T>` support implicit conversion from `T`.
```C#
Selector<string> a = "Hello World";  // Good
StringSelector b = "Hello World"; // Build error
```

Reply:
How about remove `WarningThumbImageUrlSelector` and privide `WarningThumbImageUrl` as `Selector<string>` type?
As I commented above, user can treat a `Selector<T>` type as a `T`.
```C#
// Both ways are possible to use.

// 1) The warning thumb image won't change by state.
slider.WarningThumbImageUrl = "image.png"; 

// 2)  Using this way, the warning thumb image changes by state.
slider.WarningThumbImageUrl = new Selector<string>() {
   Normal = "normal_image.png",
   Pressed = "pressed_image.png"
};
```